### PR TITLE
[CBRD-22109] Update query defaults

### PR DESCRIPTION
**Pull request by radudoros**
http://jira.cubrid.org/browse/CBRD-22109

Update queries, triggers, merge-updates on tables/views having attributes containing on update defaults should use this information to perform automatic update of that attribute, when this attribute is absent in the query.

### CODE REVIEW

**Comment by eseokoh** [comment]
@radudoros 

please find a client crash of case https://github.com/CUBRID/cubrid-testcases/blob/develop/sql/_13_issues/_12_1h/cases/bug_bts_7181.sql

I don't know the other two crashes are same as the first case. Please also investigate them.

**Comment by eseokoh** [comment]
I'd like to propose to rename `check_for_default_expr` and `check_for_on_update_expr` as `pt_find_omitted_default_expr` and `pt_generate_assignment_expr_of_default_expr` and move them to `xasl_generation.c`.
I hope you can propose better names. :)

File: src/parser/view_transform.c [hunk 1]
```diff
@@ -8816,8 +8816,12 @@ mq_class_lambda (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE * class_,
 		  spec->info.spec.entity_name = NULL;
 		}
 
-	      newspec->info.spec.range_var->info.name.original = spec->info.spec.range_var->info.name.original;
-	      newspec->info.spec.location = spec->info.spec.location;
+	      if (statement->node_type != PT_UPDATE || spec->info.spec.join_type != PT_JOIN_NONE)
+		{
```
Comment (eseokoh) [inline src/parser/view_transform.c]: please add descriptions why we need `if`.

**Comment by ribram** [comment]
@radudoros Looks great, please merge it :)

**Pull request merged**